Rebranding 1 rebased

Norrox opened the pull request:
https://github.com/blazium-engine/blazium/pull/12

flare199 commented:
Did you change an alarm log message to spanish in platform/android/java/lib/src/org/godotengine/godot/GodotDownloaderAlarmReceiver.java?

sshiiden commented:
> Did you change an alarm log message to spanish in platform/android/java/lib/src/org/godotengine/godot/GodotDownloaderAlarmReceiver.java?

I checked, it was already in spanish.

Norrox commented:
> Did you change an alarm log message to spanish in platform/android/java/lib/src/org/godotengine/godot/GodotDownloaderAlarmReceiver.java?
it was spanish original, but i changed it to english and it broke everything xD
so i changed it back :P

Bioblaze commented:
Wait what?! it was in spanish to begin with? and then broke?!?!?!
What was the error of the failed test when it broke?

Norrox commented:
> Wait what?! it was in spanish to begin with? and then broke?!?!?! What was the error of the failed test when it broke?
![image](https://github.com/user-attachments/assets/a06056c5-4d1f-4c3d-ba0c-36ee44775a24)
![image](https://github.com/user-attachments/assets/354aa0df-83f1-4e9b-b6d5-3a741bd002bb)

jss2a98aj commented on scene/resources/material.cpp:
```diff
@@ -3546,7 +3546,7 @@ bool StandardMaterial3D::_set(const StringName &p_name, const Variant &p_value)
 		}
 
-		WARN_PRINT("Godot 3.x SpatialMaterial remapped parameter not found: " + String(p_name));
+		WARN_PRINT("Blazium 3.x SpatialMaterial remapped parameter not found: " + String(p_name));
```
There is no Blazium 3.x as of right now, this should probably be left as "Godot 3.x"

Norrox replied:
fixed https://github.com/blazium-engine/blazium/pull/30/commits/a11e1f011a19a6fab53b6533e424d162f39339fb

jss2a98aj commented on platform/web/export/export_plugin.cpp:
```diff
@@ -387,7 +387,7 @@ Ref<Texture2D> EditorExportPlatformWeb::get_logo() const {
 #ifdef MODULE_MONO_ENABLED
 	// Don't check for additional errors, as this particular error cannot be resolved.
-	r_error += TTR("Exporting to Web is currently not supported in Godot 4 when using C#/.NET. Use Godot 3 to target Web with C#/Mono instead.") + "\n";
+	r_error += TTR("Exporting to Web is currently not supported in Blazium 4 when using C#/.NET. Use Blazium 3 to target Web with C#/Mono instead.") + "\n";
```
This should probably be shortened to "Exporting to Web is currently not supported in Blazium 4 when using C#/.NET." since there is no Blazium 3.

Norrox replied:
fixed https://github.com/blazium-engine/blazium/pull/30/commits/25d9fd1d0892b1650c16536d037320d2d6e69271

jss2a98aj commented on platform/android/java/scripts/publish-module.gradle:
```diff
@@ -21,6 +21,7 @@ afterEvaluate {
                     name = PUBLISH_ARTIFACT_ID
                     description = 'Blazium Engine Android Library'
                     url = 'https://blazium.app/'
+
```
This file doesn't need modified just to add a new line.

jss2a98aj commented on platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:
There seem to be non-rebranding changes in this file.

jss2a98aj commented on platform/android/java/lib/patches/com.google.android.vending.licensing.patch:
There seem to be non-rebranding changes in this file too.

Norrox commented:
Think this is ready for merge when the tests are good?
